Formula/circleci.rb

```diff
@@ -19,10 +19,13 @@ def install
     ENV["GOPATH"] = buildpath
     ENV["GO111MODULE"] = "on"
 
+    system "go get -u github.com/gobuffalo/packr/v2/packr2"
```

Comment:
This is not how we handle Go dependencies. Please look at other vending strategies in similar Go formulas.